e2e/internal/e2e/singularitycmd.go

```diff
@@ -411,16 +414,30 @@ func RunSingularity(t *testing.T, cmdPath string, cmdOps ...SingularityCmdOp) {
 		t.Errorf("ExpectExit is missing in cmdOps argument")
 		return
 	}
+	if s.profile == 0 {
+		availableProfiles := make([]string, len(Profiles))
+		for i, p := range Profiles {
+			availableProfiles[i] = p.String()
+		}
+		profiles := strings.Join(availableProfiles, ", ")
+		t.Errorf("you must specify a profile, available profiles are %s", profiles)
+		return
+	}
+	privileged := s.profile.privileged()
```

Comment:
This default is only because E2E is executed under a `sudo` command, right?

Reply:
`sudo` is used once when running `sudo go test ...`, after if `privileged` is `true` singularity is executed with root identity via a dedicated privileged thread by calling `e2e.Privileged()`